docs/tour/adding_capabilities.mdx

```diff
@@ -0,0 +1,232 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Adding Capabilities
+
+Going from "Hello World" to a full-fledged application requires just identifying what capabilities your application needs, and then adding them to your actor. Applications in wasmCloud have two types of capabilities: external requests that call a function handler in your actor (like HTTP Server and Messaging) and actor requests that call a function handler in a capability provider (like Key-Value Store and Logging). "Hello World" already has the HTTP Server capability, so let's add more features to this application with functional code and more capabilities.
+
+## Adding more functionality
+
+Let's extend this application to do more than just say "Hello World". We can use the `HttpRequest` struct and check the request for a name provided in a query string, and then return a greeting with that name.
+
+```rust
+async fn handle_request(&self, ctx: &Context, req: &HttpRequest) -> RpcResult<HttpResponse> {
+    let name = match req.query_string.split("=").collect::<Vec<&str>>()[..] {
+        ["name", name] => name.to_string(),
+        _ => "World".to_string(),
+    };
+
+    Ok(HttpResponse::ok(format!("Hello, {}!", name,)))
+}
+```
+
+Using a simple pattern match, we can check to see if we can split the query string on a name parameter, and if so, use that name in the response. If not, we'll just use "World" as the name. This won't support adding multiple query parameters but it's a perfectly good start for our goals.
```

Comment:
maybe just include some sample URLs illustrating the URLs that will produce the 2 different outcomes. If I'm a Rust newb, that pattern match may not be self-documenting.